core/logger/logger.go

```diff
@@ -92,9 +92,11 @@ type Logger interface {
 
 // Constants for service names for package specific logging configuration
 const (
-	HeadTracker = "head_tracker"
-	FluxMonitor = "fluxmonitor"
-	Keeper      = "keeper"
+	HeadTracker  = "head_tracker"
+	HeadListener = "head_listener"
+	HeadSaver    = "head_saver"
+	FluxMonitor  = "fluxmonitor"
+	Keeper       = "keeper"
```

Comment:
Should we consider aligning this format with log message prefixes and `Named()` loggers? 
```suggestion
	HeadTracker  = "HeadTracker"
	HeadListener = "HeadListener"
	HeadSaver    = "HeadSaver"
	FluxMonitor  = "FluxMonitor"
	Keeper       = "Keeper"
```
These are persisted though IIRC...

Reply:
(FWIW only `head_tracker` is used)